darwin/dataset/utils.py

```diff
@@ -130,7 +131,7 @@ def make_class_lists(release_path: Path):
 
 
 def get_classes(
-    dataset_path: Path,
+    dataset_path: Union[Path, str],
```

Comment:
Are we sure we need `str` here? I see you're rightly doing `dataset_path = Path(dataset_path)`

Reply:
Since I'm also allowing `str` (because I'm immediately converting it to `Path`) don't I need to specify that in the header?